### ISSUE TITLE
QandAの質問文の追加のお願い

### ISSUE THREAD

**Issue by yamatomatsuda** [l1sum/sound_system#10]
@hir-osechi 
受け取った人数を答える質問文を追加してほしいです。（つまり、動的に質問文を変える必要がある）
`            if "QandA" == command[0].replace("Command:",""):
                if module_QandA.QandA(command[1].replace("Content:",""))==1:
                    self.cerebrum_publisher("Retern:0,Content:None")
`
のContentに認識した男女の人数を送ります。
したがって想定している受信されるメッセージは(Subscriberで購読するメッセージは)

`Command:QandA,Content:4|5`

のようになります。
**branchはfeature_modify_for_launchをpullする専用のブランチで作業をお願いします**

**Comment by hir-osechi** [comment]
module_QandA.QandA(command[1].replace("Content:",""))==1:っていうのは、returnで1を返されたらってこと？

**Comment by yamatomatsuda** [comment]
ごめんそれはミス
メッセージが
''
Command:QandA,Content:4|5
''
みたいに送られてくるので

男女それぞれ何人？と聞くと、

男4，女5人です。と答えてほしいわけです。
したがって、受けとった人数を回答文に反映させればいいのだけど、できそう？

**Comment by hir-osechi** [comment]
@yamatomatsuda 
QandAの()内に要素入れるとしたら、普通の質疑応答の時もなにか入れないといけないよね？
それかpersin recognitionと回転して質疑応答の時のQ/Aのモジュール変える？

**Comment by yamatomatsuda** [comment]
ちょっと普通が何言ってるかわからないけど、
競技的には、開始合図の発話ー＞回転ー＞群衆認識ー＞質疑応答ー＞音源定位の順番だよね
この群衆認識で得たデータを受け取る
それが
`Command:QandA,Content:4|5`
のような形でおくられてくる
それを次の質疑応答で使いたいってことなんだけど

**Comment by hir-osechi** [comment]
うん、群衆認識の質疑応答5回と、その後に画像処理を伴わない普通の質疑応答5回があると思うんだけど、後者のときってQandAの()の中になにか入れる？

今現状は、()に何も入れずに、このモジュールを呼ぶと質問を認識して答えを発話を5回したら終了ってなってる

**Comment by yamatomatsuda** [comment]
なるほどね、なら可変長のargs使ってkeyword認識をしてくれたら助かるね
()の中が何もないと、5回繰り返す
(human=4)みたいに指定すると、特定の質問文の認識になる
男女の人数が与えられた場合も同じような形にしよう
https://qiita.com/raviqqe/items/ee2bcb6bef86502f8cc6
ここを参考に作ってみてくれ！
